cmd/gaia/test_cli/cli_test.go

```diff
@@ -0,0 +1,290 @@
+package clitest
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/client/keys"
+	"github.com/cosmos/cosmos-sdk/cmd/gaia/app"
+	gaiacli "github.com/cosmos/cosmos-sdk/cmd/gaia/cmd/gaiacli/gaiaclicmd"
+	gaiad "github.com/cosmos/cosmos-sdk/cmd/gaia/cmd/gaiad/gaiadcmd"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/server"
+	"github.com/cosmos/cosmos-sdk/tests"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	stakeTypes "github.com/cosmos/cosmos-sdk/x/stake/types"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	keyFoo   = "foo"
+	keyBar   = "bar"
+	fooDenom = "fooToken"
+	bdenom   = stakeTypes.DefaultBondDenom
+)
+
+var fooCoins = sdk.Coins{
+	sdk.NewCoin(bdenom, sdk.NewInt(150)),
+	sdk.NewCoin(fooDenom, sdk.NewInt(1000)),
+}
+
+func TestGaiaCLIMinimumFees(t *testing.T) {
+	// t.Parallel()
+	f := InitializeFixtures(t)
+
+	// start gaiad server with minimum fees
+	go f.GDStart("--minimum_fees=2feeToken")
+
+	tests.WaitForTMStart(f.RPCPort)
+	tests.WaitForNextNBlocksTM(1, f.RPCPort)
+
+	fooAddr := f.GCLIKeysAddress(keyFoo)
+	barAddr := f.GCLIKeysAddress(keyBar)
+
+	fooAcc := f.GCLIGetAccount(fooAddr)
+	require.Equal(t, int64(50), fooAcc.GetCoins().AmountOf(bdenom).Int64())
+
+	bytes := f.GCLISend(keyFoo, barAddr, sdk.Coins{sdk.NewCoin(bdenom, sdk.NewInt(10))})
```

Comment:
I'm not entirely clear what we're testing here - shouldn't we test that a tx is rejected with too few fees and accepted with enough?